platforms/documentation/docs/src/docs/userguide/authoring-builds/introduction/build_lifecycle_intermediate.adoc

```diff
@@ -0,0 +1,116 @@
+- Creates a link:{groovyDslPath}/org.gradle.api.initialization.Settings.html[`*Settings*`] instance +
+- Evaluates the *settings file* to determine which projects (and included builds) make up the build +
+- Creates a link:{groovyDslPath}/org.gradle.api.Project.html[`*Project*`] instance *for every project*
+
+| - Evaluates the *build files of every project* participating in the build +
+- Creates a *task graph* for requested tasks
+
+| - Schedules and *executes* the selected *tasks*
+|===
+
+image::build-lifecycle-example.png[]
+
+The following example shows which parts of settings and build files correspond to various build phases:
+
+====
+include::sample[dir="snippets/buildlifecycle/basic/kotlin",files="settings.gradle.kts[];build.gradle.kts[]"]
+include::sample[dir="snippets/buildlifecycle/basic/groovy",files="settings.gradle[];build.gradle[]"]
+====
+
+The following command executes the `test` and `testBoth` tasks specified above.
+Because Gradle only configures requested tasks and their dependencies, the `configured` task never configures:
+
+[source.multi-language-sample,kotlin]
+----
+> gradle test testBoth
+include::{snippetsPath}/buildlifecycle/basic/tests/buildlifecycle.out[]
+----
+[source.multi-language-sample,groovy]
+----
+> gradle test testBoth
+include::{snippetsPath}/buildlifecycle/basic/tests/buildlifecycle.out[]
+----
+
+[[sec:initialization]]
+=== Phase 1. Initialization
+
+In the *initialization phase*, Gradle detects the set of projects (root and subprojects) and included builds participating in the build.
+
+Gradle first evaluates the settings file, `settings.gradle(.kts)`, and instantiates a `Settings` object.
+
+Then, Gradle instantiates `Project` object instances for each project included in the build (using `includeBuild()` or `include()` in the settings file).
+
+[[sec:configuration]]
+=== Phase 2. Configuration
+
+In the *configuration phase*, Gradle adds tasks and other properties to the projects found by the initialization phase.
+
+Gradle constructs the <<#sec:task_graph,task graph>> by understanding the dependencies between tasks.
+
+[[sec:project_evaluation]]
+[[sec:execution]]
+[[sec:task_execution]]
+=== Phase 3. Execution
+
+In the *execution phase*, Gradle runs tasks.
+
+Gradle uses the <<#sec:task_graph,task execution graphs>> generated by the configuration phase to determine which tasks to execute.
+Gradle can execute tasks in parallel.
+
+[[sec:task_graph]]
+== Task Graphs
+
+As a build author, you write build logic by defining tasks and declaring how they depend on one another.
+Gradle uses this information to construct a **task graph** during the *configuration phase* that models the relationships between these tasks.
+
+For example, if your project includes tasks such as `build`, `assemble`, and `createDocs`, and you declare that `assemble` depends on `build`, and `createDocs` depends on `assemble`, Gradle constructs a graph with this order: `build` → `assemble` → `createDocs`.
```

Comment:
There are tasks called "assemble" and "build, and in practice "build" depends on "assemble", perhaps change the example to match that?